pyomo/scripting/driver_help.py

```diff
@@ -16,8 +16,9 @@
 import textwrap
 import logging
 import socket
+import subprocess
 
-import pyutilib.subprocess
+import pyutilib.misc
```

Comment:
The `import pyutilib.subprocess` was sufficient to allow access to `pyutilib.misc.sort_index` later in the file. Since I removed the `pyutilib.subprocess` import, I needed to add back the `pyutilib.misc` (at least until I rip those out later).